mslib/msui/mpl_map.py

```diff
@@ -145,10 +144,8 @@ def __init__(self, identifier=None, CRS=None, BBOX_UNITS=None, PROJECT_NAME=None
                     self.crs_text = self.ax.figure.text(0, 0, self.crs)
 
         if self.appearance["draw_graticule"]:
-            try:
```

Comment:
@joernu76 please look on this change

Reply:
@ReimarBauer Yes, the test was failing. Hence, it was needed. Both are related. Since, I removed `_draw_auto_graticule()` from def __init__() of mpl_map because this is already being called in __init__() of  MPLTopview Class of mpl_qtwidget(since it is necessary to draw graticule with the already changed fontsizes). And hence, since I removed this `_draw_auto_graticule()` from mpl_map.py, the test also needed to be changed otherwise the test would continue to fail.

Reply:
ok, the test is likly now on the"wrong" place, but we can move it later. 
We do avoid this when the test is using MPLTopview